docs/chapters/architecture/DLT/diagnostics-log-and-tracing.rst

```diff
@@ -0,0 +1,181 @@
+:orphan:
+
+.. _diagnostics-log-and-tracing:
+
+
+Diagnostics Log and Tracing
+===========================
+
+Introduction and goals
+----------------------
+The Diagnostic, Log and Tracing (DLT) daemon provides a log and trace interface in ECUs. The standardized protocol for logging inside ECU is specified in the AUTOSAR 4.0 DLT, and GENIVI has an open source project `dlt-daemon`_ that follows those standards.
+
+Besides dlt-daemon, GENIVI offers `dlt-viewer`_ which is another software that allows a client to connect to the dlt-daemon and receive logs remotely.
+
+PELUX has decided to integrate GENIVIs' DLT component which also works with dlt-viewer.
+
+Requirements overview
+^^^^^^^^^^^^^^^^^^^^^
+Log and Trace
+"""""""""""""
+DLT logs are targeted at OEM-s (developers, testers), since they show technical information and give an overview of the processes. During the development, it is entirely up to the developers to decide what to store on the DLT logs.
+
+On a PELUX image, the dlt-daemon package is already integrated and enabled.
+
+In order to get DLT logs remotely, `dlt-viewer`_ is required.
+
+Basic usage
+"""""""""""
+The dlt-daemon runs by default when the PELUX image boots up. dlt-daemon usage can be found on https://github.com/GENIVI/dlt-daemon.  Cheat-sheets and further documentation can be found on https://github.com/GENIVI/dlt-daemon/tree/master/doc (Although the documentation here is outdated, it can be very helpful giving insight into architecture).
```

Comment:
Is there a way to make "dlt-daemon" and "dlt-viewer" a certain format/font so that it's obvious they are names of components?

Reply:
Actually I think there are instructions somewhere on how we write this. Somewhere on the top level of swf or something. I think component may be italic.